modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -389,6 +401,8 @@ if (CONFIG_BUILD_WITH_TFM)
     if(PAD)
       set(pad_args --pad --pad-header)
     endif()
+    string(REPLACE "\${CONFIG_TFM_MCUBOOT_SIGNATURE_TYPE}" "${CONFIG_TFM_MCUBOOT_SIGNATURE_TYPE}"
+                    CONFIG_TFM_KEY_FILE_${SUFFIX} ${CONFIG_TFM_KEY_FILE_${SUFFIX}})
```

Comment:
Are `CONFIG_TFM_KEY_FILE_S` and `CONFIG_TFM_KEY_FILE_NS` valid symbols ? Isn't TFM using `TFM_KEY_S` and `TFM_KEY_NS` ?